### PR TITLE
[13.0][IMP] logistics_planning_*: some improvements

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
- logistics_planning_mgmt_weight: vehicle_type_id editable in ready state
- logistics_planning_base: editable commitment_date and logistics_price_unit_done fields in draft state
- logistics_planning_base: new field rescheduled_date

### CODE REVIEW

**Comment by dalonsod** [comment]
Code review :+1: 
@ChristianSantamaria please make a rebase (once #154 is recently merged) and install these new improvements in AluTest environment.

**Comment by ChristianSantamaria** [comment]
Rebased and updated in test

**Comment by dalonsod** [comment]
We won't wait for customer approve, if a change is needed, we'll open a new PR... merging!

**Review by dalonsod — APPROVED** [review]

**Pull request merged**
